src/text/skeleton.scss

```diff
@@ -3,21 +3,26 @@
 @import '../style/index';
 
 @mixin iui-text-skeleton {
-  display: inline-block;
   user-select: none;
   color: transparent;
   border-radius: $iui-border-radius;
+  cursor: wait;
 
   @include themed {
     background: linear-gradient(
-      270deg,
-      t(iui-color-background-3),
-      t(iui-color-background-3),
-      t(iui-color-background-2),
-      t(iui-color-background-3),
-      t(iui-color-background-3)
+      292deg,
+      t(iui-text-color-muted),
+      t(iui-text-color-muted),
+      t(iui-text-color-placeholder),
+      t(iui-text-color-muted),
+      t(iui-text-color-muted)
     );
-    background-size: 200% 100%;
+  }
+  background-size: 200% 100%;
```

Comment:
It is expected that skeleton is same width on all elements, no matter the content?
![image](https://user-images.githubusercontent.com/81580355/163543206-05eda701-ba64-46ee-9fed-f638772d4170.png)

Reply:
That is discussed [here](https://github.com/iTwin/iTwinUI/pull/587#discussion_r849120409).  You can set `width: fit-content` to match the width of the text (if text is present) or you can set `width: 30%` like Vilius did [here](https://github.com/iTwin/iTwinUI/blob/f6657791d42d6818760728615ae4cc45f4a0edb9/src/menu/menu.scss#L155).

Reply:
I thought you will just revert your change 😀

Reply:
What?
But now its worse. If I return `inline-block` skeletons take the width of the text, and thats what we want.
Then if users want to have skeleton without actual content (which is often the case), they can set the width (or flex container, which should grow children, i think?)

Reply:
If I set `display: inline-block;` I'm getting this:
![Screen Shot 2022-04-15 at 11 21 29 AM](https://user-images.githubusercontent.com/849817/163588940-0e82597d-9866-4e7c-a6d2-dad486d19512.png)
 
It doesn't seem right to override the `display` when skeleton is set in my opinion.

Reply:
Put `display: inline-block` back & set `section[id^='demo'] { display: flex; flex-direction: column; }`.